.github/workflows/release-workflow.yml

```diff
@@ -0,0 +1,53 @@
+
+name: Release Trace Analytics Artifacts
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+
+      - name: Checkout Kibana
+        uses: actions/checkout@v1
+        with:
+          repository: opendistro-for-elasticsearch/kibana-oss
+          ref: 7.10.0
+          token: ${{secrets.OD_ACCESS}}
+          path: kibana
+
+      - name: Checkout Plugin
+        uses: actions/checkout@v1
+        with:
+          path: kibana/plugins/trace-analytics
+
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '10.22.1'
+
+      - name: Kibana Pluign Bootstrap
+        run: |
+          yarn kbn bootstrap
+
+      - name: Build Artifact
+        run: |
+          yarn build
+          mv ./build/*.zip ./build/opendistroTraceAnalyticsKibana-1.12.0.0.zip
```

Comment:
maybe use a env variable for version number and name?

Reply:
good idea, will update